ros/openapi/openapi.json

```diff
@@ -537,7 +540,7 @@
                 "name": "limit",
                 "in": "query",
                 "required": false,
-                "description": "Number of items to return, set -1 to return all",
+                "description": "Number of items to return",
```

Comment:
@kgaikwad I hope this is okay for now, as keeping that text would mean we will need code changes.
-1 is not returning systems at the moment.

Reply:
Make sense!